graylog2-web-interface/src/components/common/LinkToNode.jsx

```diff
@@ -15,18 +15,18 @@
  * <http://www.mongodb.com/licensing/server-side-public-license>.
  */
 import PropTypes from 'prop-types';
-import React from 'react';
-import createReactClass from 'create-react-class';
-import Reflux from 'reflux';
+import * as React from 'react';
 
 import { Link } from 'components/common/router';
 import Routes from 'routing/Routes';
 import AppConfig from 'util/AppConfig';
+import HideOnCloud from 'util/conditional/HideOnCloud';
 import { NodesStore } from 'stores/nodes/NodesStore';
 
 import Icon from './Icon';
 import Spinner from './Spinner';
 
+import connect from '../../stores/connect';
```

Comment:
We have aliases so for import you can use this below. Then you will need to move the line before the Icon import.
```suggestion
import connect from 'stores/connect';
```